pytest_docker_tools/wrappers/fixture_ref.py

```diff
@@ -0,0 +1,10 @@
+class FixtureRef:
+    def __init__(self, name):
+        self._name = name
+
+    @property
+    def name(self):
+        return self._name
+
+
+fixtureref = FixtureRef
```

Comment:
We should probably think about this name some more. I pulled it out of my ass for the example in the ticket, but e.g. it's not passing a reference to a fixture at all. In some ways I think `fixtureval` is a better name (it causes the fixture to be lazily resolved and the value to be used). Though it would then end `eval` 😆

I also don't want it to be too long.

Can you think of anything better?

I also thought about going with `F` or `f` but thats probably going too far. I also thought about `lazyfixture` but it's long, and understanding how and why it's "lazy" is really an implementation detail the user shouldn't care about.

Reply:
Was also wondering whether to use a named tuple for the class.

Also possibly make it private. As it's an implementation detail. Then have a func that returns one. I don't want people to come up with any weird ideas about subclassing it downstream.

Reply:
Its now private and for the moment I kept the alias which is meant for end users